tasks/push-disk-images/push-disk-images.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-disk-images
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push disk images via an InternalRequest
+  params:
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+    - name: dataPath
+      type: string
+      description: Path to the data JSON in the data workspace
+    - name: pipelineRunUid
+      type: string
+      description: The uid of the current pipelineRun. Used as a label value when creating internal requests
+  workspaces:
+    - name: data
+      description: Workspace where the json files are stored
+  steps:
+    - name: run-script
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+      script: |
+        #!/bin/sh
+        set -ex
+
+        pipelinerun_label="internal-services.appstudio.openshift.io/pipelinerun-uid"
+        snapshot=$(jq -c $(workspaces.data.path)/$(params.snapshotPath))
+        # .cdn.env is likely to change in the future. This is just for POC
+        env=$(jq -er '.cdn.env' $(workspaces.data.path)/$(params.dataPath))
+
+        # There are three envs supported...production, stage, and qa
+        exodusGwSecret=""
+        exodusGwEnv=""
+        pulpSecret=""
+
+        if [ "${env}" == "production" ] ; then
+          exodusGwSecret="exodus-prod-secret"
+          exodusGwEnv="live"
+          pulpSecret="rhsm-pulp-prod-secret"
+        elif [ "${env}" == "stage" ] ; then
+          exodusGwSecret="exodus-prod-secret"
```

Comment:
not stage?

Reply:
Per https://issues.redhat.com/browse/RHELDST-24850, the exodus gw is the same for both prod and stage, we just use different pulp and env values

Reply:
maybe add a comment so ppl don't ask same question at a later time?

Reply:
added